Steps4Impact/DashboardV2/DashboardDataSource.swift

```diff
@@ -74,8 +74,17 @@ class DashboardDataSource: TableViewDataSource {
         self.eventTimelineString = DataFormatters
           .formatDateRange(value: (start: event.challengePhase.start, end: event.challengePhase.end))
         self.eventLengthInDays = event.lengthInDays
+        self.commitment = event.commitment?.miles ?? 300
+      } else {
+        self.eventName = "No Current Event"
+        let monthsInSeconds: Double = 60 * 60 * 24 * 30
```

Comment:
Should we figure out the time in the month using NSCalendar rather than assuming 30 days?